pvlib/spectrum/mismatch.py

```diff
@@ -710,3 +710,86 @@ def spectral_factor_jrc(airmass, clearsky_index, module_type=None,
         + coeff[2] * (airmass - 1.5)
     )
     return mismatch
+
+
+def spectral_factor_polo(precipitable_water, airmass_absolute, aod500, aoi,
+                         altitude, module_type=None, coefficients=None,
+                         albedo=0.2):
+    """
+    Estimation of spectral mismatch for BIPV application in vertical facades.
```

Comment:
```suggestion
    Estimate the spectral mismatch for BIPV application in vertical facades.
```
A suggestion, I think the imperative form is standard practice for the summary line